### PR TITLE
build-sync: Emergencies only for stream

### PR DESCRIPTION
**Pull request by joepvd**
See https://github.com/openshift-eng/art-tools/pull/69

### CODE REVIEW

File: jobs/build/build-sync/Jenkinsfile [hunk 1]
```diff
@@ -56,9 +56,10 @@ node('covscan') {
                 ),
                 booleanParam(
                     name        : 'EMERGENCY_IGNORE_ISSUES',
-                    description : ['Ignore all issues with constructing payload. ',
-                                   'In gen-payload, viable will be true whatever is the case, ',
-                                   'making internal consistencies in the nightlies possible.<br/>',
+                    description : ['Ignore all issues with constructing payload.',
+                                   'In gen-payload, viable will be true whatever is the case,',
+                                   'making internal consistencies in the nightlies possible.',
```
Comment (locriandev) [inline jobs/build/build-sync/Jenkinsfile:61]: Shouldn't that be making internal `inconsistencies`?

Comment (joepvd) [reply jobs/build/build-sync/Jenkinsfile]: 👍🏼

**Review by locriandev — COMMENTED** [review]
/lgtm

**Pull request merged**